pretext/ExtraTopics/More-BigO.ptx

```diff
@@ -0,0 +1,160 @@
+<?xml version="1.0"?>
+<section xml:id="extratopics-more-bigo">
+  <title>Big O Simplification and Practice</title>
+
+  <p>One more important topic in Big O Analysis is <term>simplification</term>. You will often be asked to do
+    this on assignments. We won't get too deep into the math, but here are a few general rules:</p>
+
+  <p>Constant time operations always simplify to <c>O(1)</c>. You <em>can't</em> have complexities like <c>O(3)</c> or <c>O(1000000)</c>.</p>
+
+  <p>Added constants are ignored when a larger factor is around. For instance, <c>O(n + 1)</c> is just <c>O(n)</c>.</p>
+
```

Comment:
Oh, thanks for reminding me! I felt like I was missing one

Reply:
Yeah - this is where I think using the limits might be helpful. Bc when you take a limit at infinity, you can kinda ignore the less significant terms. 

I think it might be helpful if we explain what the significance of the terms are.